packages/clerk-js/src/ui/components/PricingTable/PricingTableDefault.tsx

```diff
@@ -175,16 +184,53 @@ function Card(props: CardProps) {
             />
           </Box>
         ) : null}
+
+        {showStatusRow && (
+          <Box
+            sx={t => ({
+              padding: t.space.$1,
+              borderTopWidth: t.borderWidths.$normal,
+              borderTopStyle: t.borderStyles.$solid,
+              borderTopColor: t.colors.$neutralAlpha100,
+            })}
+          >
+            {subscription?.status === 'active' || (isImplicitlyActiveOrUpcoming && subscriptions.length === 0) ? (
+              <Text
+                elementDescriptor={descriptors.pricingTableCardBadge}
+                variant='caption'
+                colorScheme='body'
+                localizationKey={localizationKeys('badge__currentPlan')}
+                sx={{ textAlign: 'center' }}
+              />
+            ) : (
+              <Text
+                elementDescriptor={descriptors.pricingTableCardBadge}
+                variant='caption'
+                colorScheme='body'
+                localizationKey={
+                  subscription
+                    ? localizationKeys('badge__startsAt', {
+                        date: subscription.periodStart,
+                      })
+                    : localizationKeys('badge__upcomingPlan')
+                }
+                sx={{ textAlign: 'center' }}
+              />
+            )}
+          </Box>
+        )}
+
         {(!plan.isDefault || !isDefaultPlanImplicitlyActiveOrUpcoming) && (
           <Box
             elementDescriptor={descriptors.pricingTableCardAction}
             sx={t => ({
               marginTop: 'auto',
               padding: isCompact ? t.space.$3 : t.space.$4,
-              borderTopWidth: hasFeatures ? t.borderWidths.$normal : 0,
+              borderTopWidth: t.borderWidths.$normal,
               borderTopStyle: t.borderStyles.$solid,
               borderTopColor: t.colors.$neutralAlpha100,
               background: undefined,
+              order: ctaPosition === 'top' ? -1 : undefined,
```

Comment:
This looks a bit off with ctaPosition as top.

![Screenshot 2025-05-04 at 7 07 18 PM](https://github.com/user-attachments/assets/d0cc3348-abf8-454f-b927-c48062ae18e9)